starlite/handlers.py

```diff
@@ -68,7 +68,8 @@ def resolve_dependencies(self) -> Dict[str, Provide]:
         """
         Returns all dependencies correlating to handler function's kwargs that exist in the handler's scope
         """
-        assert self.signature_model, "resolve_dependencies cannot be called before a signature model has been generated"
+        if not self.signature_model:
+            raise RuntimeError("resolve_dependencies cannot be called before a signature model has been generated")
```

Comment:
I'm not sure RuntimeError is correct here... but I have no better idea.

Reply:
Can you explain how this exception gets triggered? :slightly_smiling_face:

Reply:
What do you mean?

Reply:
nvm I figured it out :+1: